core-web/libs/data-access/src/lib/dot-contentlet/dot-contentlet.service.ts

```diff
@@ -26,4 +26,19 @@ export class DotContentletService {
             })
             .pipe(take(1), pluck('entity', 'versions', language));
     }
+
+    /**
+     * Get the Contentlet versions by the inode.
+     *
+     * @param string inode
+     * @returns Observable<DotCMSContentlet>
+     * @memberof DotContentletService
+     */
+    getContentletByInode(inode: string): Observable<DotCMSContentlet> {
+        return this.coreWebService
```

Comment:
We deprecated coreWebService.

Reply:
what is the new way? fetch

Reply:
Use the default httpClient of Angular.
Ex.
```
getFormIndetifier(containerId: string, formId: string): Observable<string> {
        return this.http
            .get<{ entity: { content: { idenfitier: string } } }>(
                `/api/v1/containers/form/${formId}?containerId=${containerId}`
            )
            .pipe(pluck('entity', 'content', 'identifier'));
    }

Reply:
Done here https://github.com/dotCMS/core/pull/27663/commits/f99abaa010c409d6185f0849715c75e06bf76a2c, Thanks for the feedback!